diff -Naur patchutils-0.3.1/src/filterdiff.c patchutils-0.3.1.tpg/src/filterdiff.c
--- patchutils-0.3.1/src/filterdiff.c	2009-01-23 17:15:49.000000000 +0000
+++ patchutils-0.3.1.tpg/src/filterdiff.c	2009-01-25 16:19:42.000000000 +0000
@@ -1246,7 +1246,7 @@
 			char errstr[300];
 			regerror (err, &regex[num_regex - 1], errstr,
 				  sizeof (errstr));
-			error (EXIT_FAILURE, 0, errstr);
+			error (EXIT_FAILURE, 0, "%s", errstr);
 			exit (1);
 		}
 	}
@@ -1492,7 +1492,7 @@
 			char errstr[300];
 			regerror (err, &regex[num_regex - 1], errstr,
 				  sizeof (errstr));
-			error (EXIT_FAILURE, 0, errstr);
+			error (EXIT_FAILURE, 0, "%s", errstr);
 			exit (1);
 		}
 	}
